README.md

```diff
@@ -30,15 +30,15 @@ docker-compose up -d
 docker exec -it reliefsupports_php_1 bash
 composer update
 
-chmod 777 -R storage/
+chmod -R 777 storage/
```

Comment:
This is alright for now , but you can change this to run `chown on the web servers username and group and storage access won't be a problem